src/main/frontend/db/conn.cljs

```diff
@@ -68,30 +66,13 @@
   [repo]
   (swap! conns dissoc (datascript-db repo)))
 
-(defn me-tx
```

Comment:
I removed me from all the starts since none of these `:me/*` attributes are used anymore